site/public/js/ta-grading-v2.js

```diff
@@ -93,16 +92,29 @@ $(function () {
       return;
     }
     const isPanelOpen = $('#' + panelId).is(':visible') && btnCont.hasClass('active');
-    // If panel is not in-view and two-panel-mode is enabled show the drop-down to select position,
+    // If panel is not in-view and two/three-panel-mode is enabled show the drop-down to select position,
     // otherwise just toggle it
-    if (isPanelOpen || !(taLayoutDet.isTwoPanelsEnabled && !isMobileView)) {
+    if (isPanelOpen || +taLayoutDet.numOfPanelsEnabled === 1) {
```

Comment:
What is the `+` doing here?

Reply:
Because of using LocalStorage for saving/ retrieving the variable, it can be converted to a string. So I used it to convert the type of numOfPanelsEnabled from string to number... it's just the alternative of like number(taLayoutDet.numOfPanelsEnabled)